### PR TITLE
Plug-in wcf to Maestro.

### PR DESCRIPTION
**Pull request by StephenBonikowsky**
@dagood Could you please review.

FYI
@roncain @zhenlan @iamjasonp @mconnew @hongdai

### CODE REVIEW

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -135,6 +141,26 @@
...
           ]
+        },
+        // This handler will bring the Latest CoreFX master build into WCF master
+        {
+          "maestroAction": "wcf-general",
+          "maestroDelay": "00:10:00",
+          "ScriptFileName": "run.cmd",
+          "Arguments": [
+            "build",
+            "-Project='tests\\build.proj'",
```
Comment (dagood) [inline Maestro/subscriptions.json]: This line isn't needed for wcf--it looks like you used the coreclr one as an example, but corefx and coreclr are actually different.

**Comment by dagood** [comment]
Other than that `-Project` line this looks good to me. I checked out the VSTS def and it seemed good.

File: Maestro/subscriptions.json (hunk 1 above)
Comment (StephenBonikowsky) [inline Maestro/subscriptions.json]: I removed that line in the three places where I added a Wcf handler.

**Comment by StephenBonikowsky** [comment]
FYI, I can't merge, please merge for me once it looks good.

File: Maestro/subscriptions.json (hunk 1 above)
Comment (dagood) [inline Maestro/subscriptions.json]: I forgot, another difference is that there is no `run build` in the wcf config.json--only the coreclr one has that. It's `build-managed.cmd` in wcf and corefx. (Aka `run build-managed`, although to be consistent the batch file would be better here.)

Comment (StephenBonikowsky) [inline Maestro/subscriptions.json]: Got it.

**Comment by dagood** [comment]
Looks good now, thanks! Merging.

**Pull request merged**